4_model_evaluation.yml

```diff
@@ -0,0 +1,55 @@
+target_default: 4_model_evaluation
+
+packages:
+  - dplyr
+  - scipiper
+
+sources:
+  - code/Metric_fetch.R
+  - code/Metrics_process.R
+  - code/Metric_visualize.R
+  - code/Metric_calculate.R
+
```

Comment:
I don't completely follow your file names.

I think the file that contains the metric functions should be something like `model_evaluation_functions.R` or something similar. The file that _applies_ the functions should be something like `evaluate_models.R`. This helps me understand where you're storing standard functions, and where you're applying those functions. This is more of an art and a subjective thing that any hard and fast rules.